kai/models/kai_config.py

```diff
@@ -46,7 +46,8 @@ class KaiConfigIncidentStoreInMemory(BaseModel):
 class KaiConfigModels(BaseModel):
     provider: str
     args: dict
-    template: str = Field(default="")  # TODO: Hack until we get better templating
+    template: str = Field(default="main.jinja")
```

Comment:
Do you think we could default to `{model-id}.jinja` with a fallback to main? That way we can have dedicated model-specific templates without swapping out configs.